### PR TITLE
totals and date stats on pipeline page

### PR DESCRIPTION
**Pull request by jdorweiler**
I'll need some help on making it look nice. @MariagraziaAlastra 
selection_019

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
@jdorweiler Wow, nice!

File: src/js/ia/Helpers.js [hunk 1]
```diff
@@ -273,4 +273,58 @@
...
+                total += values[i];
+            }
+        }
+        return total;
+    });
+
+    // return total number of IAs since a given number of days
+    // obj: array of IA objects
+    // days: int
+    // operator: 'less-than' 'greater-than'
```
Comment (MariagraziaAlastra) [inline src/js/ia/Helpers.js]: neat

File: src/js/ia/Helpers.js [hunk 2]
```diff
@@ -273,4 +273,58 @@
...
+    Handlebars.registerHelper('stats_from_day', function(obj, days, operator) {
+        var calc = function(date, operator) {
+            var days_from = function(tmpdate){
+                var dateObj = moment.utc(tmpdate, "YYYY-MM-DD");
+                var elapsed = parseInt(moment().diff(dateObj, "days", true));
```
Comment (MariagraziaAlastra) [inline src/js/ia/Helpers.js]: Good use of momentjs here

File: src/templates/dev_pipeline.handlebars [hunk 3]
```diff
@@ -1,10 +1,51 @@
...
 {{#with dev_milestones}}
+<h2>Total IAs in Pipeline: {{sum (length planning) (length testing) (length development) (length complete)}}</h2>
+
+<h2>Created Today: 
+    {{sum 
+        (stats_from_day planning    1 'less-than') 
+        (stats_from_day development 1 'less-than') 
+        (stats_from_day testing     1 'less-than') 
+        (stats_from_day complete    1 'less-than') 
+    }}
```
Comment (MariagraziaAlastra) [inline src/templates/dev_pipeline.handlebars]: :+1:

File: src/js/ia/Helpers.js (hunk 2 above)
Comment (jdorweiler) [inline src/js/ia/Helpers.js]: May have borrowed it from somewhere above :smile:

**Comment by MariagraziaAlastra** [comment]
@jdorweiler this is how it looks now:
screenshot-maria duckduckgo com 5001 2015-09-25 15-15-43

**Comment by MariagraziaAlastra** [comment]
Thanks @jdorweiler! :+1:

**Comment by jdorweiler** [comment]
:cactus:

**Pull request merged**